pkg/subctl/cmd/utils/utils.go

```diff
@@ -94,3 +99,32 @@ func GetSubmarinerResource(config *rest.Config) *v1alpha1.Submariner {
 
 	return submariner
 }
+
+// IsValidCIDR validates the supplied string is a valid v4/v6 network address
+func IsValidCIDR(cidr string) error {
+	ip, _, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return err
+	}
+	if ip.IsUnspecified() {
+		return fmt.Errorf("%s can't be unspecified", cidr)
+	}
+	if ip.IsLoopback() {
+		return fmt.Errorf("%s can't be in loopback range", cidr)
+	}
+	if ip.IsLinkLocalUnicast() {
+		return fmt.Errorf("%s can't be in link-local range", cidr)
+	}
+	if ip.IsLinkLocalMulticast() {
+		return fmt.Errorf("%s can't be in link-local multicast range", cidr)
+	}
+
+	return nil
+}
+
+// GetNetworkAddress returns the network address from a given CIDR
+func GetNetworkAddress(str string) string {
+	_, cidr, _ := net.ParseCIDR(str)
```

Comment:
I don't think we should ignore the error. As such we might as well just inline `net.ParseCIDR` as we do in other places. Either way it's recommended to avoid `utils` packages anyway and this file will be going way once the code restructure is done.